src/Microsoft.Performance.SDK.Runtime.NetCoreApp/Plugins/PluginsLoader.cs

```diff
@@ -57,9 +57,16 @@ public class PluginsLoader
         private readonly HashSet<IPluginsConsumer> subscribers;
 
         public PluginsLoader()
+            : this(new IsolationAssemblyLoader(), x => new SandboxPreloadValidator(x, VersionChecker.Create()))
+        {
+        }
+
+        public PluginsLoader(
```

Comment:
since this is an entry point to loading plugins, maybe we should have docstrings on these two ctors and explain what the params below are for